### PR TITLE
feat: add goreleaser workflow

### PR DESCRIPTION
**Pull request by fjogeleit**

### CODE REVIEW

File: .github/workflows/release.yaml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+name: goreleaser
+
+on:
+  push:
+    branches:
+      - 'main'
```
Comment (eddycharly) [inline .github/workflows/release.yaml:6]: are we going to release all commits ?

Comment (fjogeleit) [reply .github/workflows/release.yaml:6]: copied from the image flow, I can limit it to just tags

File: .github/workflows/release.yaml [hunk 2]
```diff
@@ -0,0 +1,34 @@
+name: goreleaser
+
+on:
+  push:
+    tags:
+      - '*'
```
Comment (eddycharly) [inline .github/workflows/release.yaml:6]:
```suggestion
      - 'v*'
```

File: .github/workflows/release.yaml [hunk 3]
```diff
@@ -0,0 +1,34 @@
+name: goreleaser
```
Comment (eddycharly) [inline .github/workflows/release.yaml:1]:
```suggestion
name: release
```

**Review by eddycharly — APPROVED** [review]

**Pull request merged**